### DISCUSSION TITLE
Microsoft Azure SSO issue - login button doesn't appear?

### DISCUSSION THREAD

**Discussion by jamsi** [outline/outline#2719] [General]
Hi folks, I've followed the [setup guide here](https://wiki.generaloutline.com/share/dfa77e56-d4d2-4b51-8ff8-84ea6608faa4) and restarted my container, yet the Login with microsoft button doesn't appear in the UI and isn't in the list of available authentication providers in POST auth.config

I installed Outline a few years ago, have just upgraded to the latest 0.59.0 so am wondering if it's a data issue. My Teams table has 1 row linked to a Google authentication provider (in the schema) and I'm looking to also add Microsoft.

Any ideas?

**Comment by tommoor** [comment]
Basically you can only have one auth setup per-team at the minute, but this has come up several times in the last few weeks and there is a PR that moves towards improving this in the works.

**Reply by jamsi** [reply]
Ah understand. Is there a document somewhere where I can read about how teams work? 

Is it team > has one auth & has many documents?

**Reply by tommoor** [reply]
Technically a team does support many auth methods in the database, there's just no way to set this up without manually editing the DB yet. Team has many collections, collections have many documents.

**Reply by jamsi** [reply]
Any advice on the necessary SQL insert to set this up? I'm assuming a new authentication_providers row, with a "name" value of microsoft and the same teamID as the google one?

**Reply by tommoor** [reply]
> I'm assuming a new authentication_providers row

That's right, `name=azure` and `providerId` would be the Microsoft tenant id (It is a UUID)

**Reply by jamsi** [reply]
That worked perfectly! Thank you! We've got both Google and Microsoft now working for the same team :)
